browser/brave_wallet/brave_wallet_delegate.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_WALLET_BRAVE_WALLET_DELEGATE_H_
+#define BRAVE_BROWSER_BRAVE_WALLET_BRAVE_WALLET_DELEGATE_H_
+
+#include "brave/components/brave_wallet/browser/browser_wallet_delegate.h"
+
+class BraveWalletDelegate : public BrowserWalletDelegate {
```

Comment:
nit: I would either use BraveBrowserWalletDelegate or BrowserWalletDelegateImpl for the class name

Reply:
actually I would make the superclass `BraveWalletDelegate` and the subclass `BraveWalletDelegateImpl`

Reply:
it's confusing to me to have BrowserWalletDelegate for BraveWalletService

Reply:
Sure I'll rename